README.rst

```diff
@@ -1,21 +1,27 @@
 ==========================
-Django eduNEXT Audit Model
+Django edunext audit model
 ==========================
 
-.. image:: https://github.com/eduNEXT/eox-audit-model/actions/workflows/tests.yml/badge.svg
-   :target: https://github.com/eduNEXT/eox-audit-model/actions/workflows/tests.yml
+|Maintainance Badge| |Test Badge| |PyPI Badge|
 
+.. |Maintainance Badge| image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+   :alt: Maintainance Status
+.. |Test Badge| image:: https://img.shields.io/github/actions/workflow/status/edunext/eox-audit-model/.github%2Fworkflows%2Ftests.yml?label=Test
+   :alt: GitHub Actions Workflow Test Status
+.. |PyPI Badge| image:: https://img.shields.io/pypi/v/eox-audit-model?label=PyPI
+   :alt: PyPI - Version
+   
+Eox audit model is a Django application that provides an audit model for tracking and logging changes within the Open edX platform.
+This plugin is designed to help administrators and developers maintain a comprehensive audit trail of various operations, ensuring better monitoring.
 
-.. image:: https://github.com/eduNEXT/eox-audit-model/actions/workflows/bump_version.yml/badge.svg
-    :target: https://github.com/eduNEXT/eox-audit-model/actions/workflows/bump_version.yml
+Features
```

Comment:
I'm honestly not sure about this section. I assume it's meant to highlight the key functionalities and capabilities of the plugin. I think we could improve it by adding the following info

```suggestion
### Features

- **Detailed Audit Logging**: Capture comprehensive logs of method executions, including parameters, results, and any generated logs.
- **Automatic Traceback Capture**: Automatically log traceback information if an exception occurs during method execution.
- **User Tracking**: Record the user who initiated the method, providing accountability and traceability.
- **Flexible Logging Mechanisms**: Log actions either by directly calling a method or using a decorator for convenience.
- **Customizable Notes**: Add custom notes to logs for additional context and information.
- **Comprehensive Monitoring**: Maintain an extensive audit trail for better monitoring and oversight of platform activities.
```